parsec/core/gui/files_widget.py

```diff
@@ -934,7 +934,7 @@ def _display_import_error(file_count, exceptions=None):
     def _on_fs_entry_synced_trio(self, event, id, workspace_id=None):
         self.fs_synced_qt.emit(event, id)
 
-    def _on_fs_entry_updated_trio(self, event, workspace_id=None, id=None):
+    def _on_fs_entry_updated_trio(self, event, workspace_id=None, id=None, **kwarg):
```

Comment:
`reason` is a mandatory param for the event, so better to explicitly provide it here:
```python
def _on_fs_entry_updated_trio(self, event, reason, workspace_id=None, id=None):
...
```